daemon/graphdriver/vfs/driver.go

```diff
@@ -98,6 +112,11 @@ func (d *Driver) parseOptions(options []string) error {
 			if err = d.setQuotaOpt(uint64(size)); err != nil {
 				return errdefs.InvalidParameter(errors.Wrap(err, "failed to set option size for vfs"))
 			}
+		case "xattrs":
+			if val != "i_want_broken_containers" {
```

Comment:
We discussed the naming of this, and it looks silly, but it makes it without doubt clear "don't use this... ever", so let's go for it